### ISSUE TITLE
The editor in the "Export to PDF" guide displays the image button in the toolbar but has no upload adapter

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#7921]
## 📝 Provide a description of requested docs changes

1.  Go to http://fake.ckeditor.com/ck/5/ckeditor5/build/docs/ckeditor5/latest/features/export-pdf.html.
2.  Try uploading an image by either clicking the "Insert image" button or drag&drop.

## Expected

The image is uploaded and inserted into the content.

**OR**

The button should not be visible in the toolbar if the former is impossible for whatever the reason.

## Actual

Nothing happens. An error in the console pops up:

```
filerepository-no-upload-adapter: Upload adapter is not defined. Read more: https://ckeditor.com/docs/ckeditor5/latest/framework/guides/…port/error-codes.html#error-filerepository-no-upload-adapter
```

**Comment by oleq** [comment]
It's even more confusing because there is an image in the editor content (the signature).

**Comment by oleq** [comment]
Closed in cksource/ckeditor5-internal#257.
